lib/regions.js

```diff
@@ -73,22 +73,54 @@ function buildPageTemplate(templates, page) {
     return pageTemplate;
 }
 
+/**
+ * Returns a boolean if there are two regions with the same name
+ * @param {Object} regions
+ * @param {Object} regionData
+ * @returns {boolean}
+ */
+function checkDuplicateRegionName(regions, regionData) {
+    let result = false;
+    Array.from(regions).forEach((region) => {
+        if (region.name === regionData.name) {
+            result = true;
```

Comment:
You can iterate with `for` loop and break here, if there's an impact on performance